hi-ml-azure/testazure/testazure/test_himl.py

```diff
@@ -1100,7 +1108,8 @@ def test_invoking_hello_world_config(run_target: RunTarget, use_package: bool, t
     parser_args = "parser.add_argument('-m', '--message', type=str, required=True, help='The message to print out')"
     extra_options = {
         'args': parser_args,
-        'body': 'print(f"The message was: {args.message}")'
+        'body': 'print(f"The message was: {args.message}")',
+        'display_name': current_test_name(),
```

Comment:
Yay, trailing commas! I suggest this hook: https://github.com/asottile/add-trailing-comma

Reply:
Looks good - wanna give it a go, since you're the pre-commit expert?

Reply:
Let's try the hook in a separate PR :-)